packages/core/assemblyManager/assembly.ts

```diff
@@ -361,8 +340,8 @@ async function loadAssemblyReaction(
     refNameAliasesList.forEach(refNameAlias => {
       refNameAlias.aliases.forEach(alias => {
         checkRefName(alias)
+        refNameAliases[alias] = refNameAlias.refName
```

Comment:
possibly note the full reversal to map alias to refName rather than refName->array of alias, it's kind of notable. i can't think of any  reason this wouldn't be allowed